frontend/index.html

```diff
@@ -98,40 +120,7 @@ <h4 style="font-weight: lighter;">
       </div>
     </div>
 
-    <!-- Handle Internet Explorer -->
-    <script>
-
-      if (window && window.navigator && window.navigator.userAgent) {
-
-        var ua = window.navigator.userAgent;
-        var ie = false;
-
-        var msie = ua.indexOf('MSIE ');
-        if (msie > 0) {
-            // IE 10 or older => return version number
-            ie = true;
-        }
-
-        var trident = ua.indexOf('Trident/');
-        if (trident > 0) {
-            // IE 11 => return version number
-            ie = true;
-        }
-
-        if (ie === true) {
-          document.querySelector('.notSupported').style.visibility='visible';
-          document.querySelector('.notSupportedButton').style.display = 'visible';
-        }
-
-      }
-
-    </script>
-
-
-
-    <!-- Entry point for all components-->
-    <div ui-view="main" flex layout="column"></div>
-    <app />
+    <div id="app"></div>
```

Comment:
Why is this removed? I can no longer see the internet explorer message (right is staging):
![image](https://user-images.githubusercontent.com/11945337/61949086-9a031700-afa1-11e9-87e8-8a754b6db707.png)

Reply:
We did it in a "better way." Whole browser detection/unsupported page logic was moved to `support.ts` (it's not included directly in the template), but maybe we set wrong IE or Edge versions. Which versions of IE should we mark as unsupported? Do you support Edge?

![image](https://user-images.githubusercontent.com/5683490/62291245-c1575980-b463-11e9-86e7-d76c150c55d9.png)

Reply:
We dont support any version of ie (they generally don't support webgl),  but we need to make sure whatever code you use for this msg it is visible for anything as low as ie6. we don't support edge either but it should work (ie they can use it but if they run into bugs we dont fix them..) , I can't remember what we currently do with it. I think we just don't say anyhthing for edge and let the user through.

Reply:
@carmenfan Unsupported browser screen shouldn't be displayed for each version of IE now. Could you check it? If it still doesn't work properly, let me know which version od IE and Windows you use.